profiles/AyushJain001.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/90152799/210181328-0922e8d4-bea4-4643-b1d3-186250b87d94.png",
```

Comment:
I recommend more of an image like this. What do you think? 🙂

![image](https://user-images.githubusercontent.com/51391473/210187235-4d9af402-a934-4622-8e68-44a56d4a47c8.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/210187235-4d9af402-a934-4622-8e68-44a56d4a47c8.png",
```